contents/docs/revenue-analytics/_snippets/data-warehouse-sources.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import List from 'components/List'
+
+const DataWarehouseSources = () => {
+    const platforms = [
+        {
+            label: 'Stripe',
+            title: 'Stripe',
+            url: '/docs/cdp/sources/stripe',
+            image: 'https://app.posthog.com/static/assets/stripe-5M7G5HUZ.png',
+        },
+        {
+            label: 'Chargebee',
+            title: 'Chargebee',
+            image: 'https://app.posthog.com/static/assets/chargebee-DYJTQLJ4.png',
```

Comment:
Are you positive you can use these hashes? Won't they change when we deploy a new posthog.com version? I really don't know the answer

Reply:
haha me neither, I'll upload them.